BuildingBlocks/src/BuildingBlocks.API/Serilog/SensitiveDataEnricherExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Serilog.Enrichers.Sensitive;
+
+namespace Backbone.BuildingBlocks.API.Serilog;
+public static class SensitiveDataEnricherExtensions
+{
+    private const string MASKED_DATA_PLACEHOLDER = "{maskedData}";
+
+    public static void AddSensitiveDataMasks(this SensitiveDataEnricherOptions options)
+    {
+        options.MaskValue = MASKED_DATA_PLACEHOLDER;
```

Comment:
Why don't we keep the default value?

Reply:
`{maskedData}` looks nicer to me than `**MASKED**`, especially in URLs. No other reason.

Reply:
On the other hand (and I think that's why the default is what it is) `**MASKED**` is a lot easier to spot. So I would probably rather go with that, so that we know exactly when we're encountering a masked log output.
But I leave it up to you.

Reply:
I will set the constant to `**MASKED**` so that it is very obvious the value can be changed and we still use the default value.